index.ts

```diff
@@ -354,39 +380,33 @@ class Analytics {
         // the User-Agent header (see https://fetch.spec.whatwg.org/#terminology-headers
         // and https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/setRequestHeader),
         // but browsers such as Chrome and Safari have not caught up.
-        'User-Agent':
-          typeof window === 'undefined'
-            ? `analytics-node/${version}`
-            : undefined,
-        Authorization:
-          'Basic ' + Buffer.from(`${this.writeKey}:`).toString('base64'),
+        ...(typeof window === 'undefined' ? { 'User-Agent': `analytics-node/${version}` } : null),
```

Comment:
`analytics-node/${version}` => `@expo/rudder-node-sdk/${version}`

Reply:
I changed this to expo-rudder-node-sdk since UA strings tend to be "product/version" and having a slash in the product name might cause issues.